install/omv-install.sh

```diff
@@ -23,25 +23,35 @@ msg_ok "Installed Dependencies"
 msg_info "Installing OpenMediaVault (Patience)"
 wget -qO- https://packages.openmediavault.org/public/archive.key | gpg --dearmor >"/etc/apt/trusted.gpg.d/openmediavault-archive-keyring.gpg"
```

Comment:
this will need to be changed to `"/usr/share/keyrings/openmediavault-archive-keyring.gpg"`